inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -165,6 +167,8 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
     manager.register_pass<ngraph::pass::ConvertNMS3ToNMS5>();
     manager.register_pass<ngraph::pass::ConvertNMS4ToNMS5>();
     manager.register_pass<ngraph::pass::ConvertNMSToNMSIEInternal>();
+    manager.register_pass<ngraph::pass::ConvertMulticlassNmsToMulticlassNmsIE>();
+    manager.register_pass<ngraph::pass::ConvertMatrixNmsToMatrixNmsIE>();
```

Comment:
Can we add SLT to demonstrate that plugin supports these operations?

Reply:
The CPU implementation of these OPs depend on these transformations and its unit-test will cover the codes. As current PR has no suitable test case cover the codes I will move all 'register_pass' to the next CPU implementation PR.